lib/dfu.js

```diff
@@ -295,5 +299,54 @@ var that = module.exports = {
 		);
 	},
 
+	_udevRulesDir: '/etc/udev/rules.d/',
+
+	_missingDevicePermissions: function (stderr) {
+		return stderr && stderr.indexOf("Cannot open DFU device") >= 0;
```

Comment:
I'd go with no because I don't know when that situation would happen. If the rules are installed and the permissions are still denied, what do you say to the user? Sorry?